packages/apps/spaces/app/src/components/EmailMultiCreatableSelect/MultiValueWithActionMenu.tsx

```diff
@@ -117,28 +125,98 @@ export const MultiValueWithActionMenu: FC<MultiValueWithActionMenuProps> = ({
     );
   };
 
+  if (showEditInput) {
+    const handleChangeValue: ReactEventHandler<HTMLElement> = (event) => {
+      const newValue = value.map((e: SelectOption<string>) =>
+        e.value === rest?.data?.value
+          ? {
+              label: (event?.target as HTMLInputElement)?.value,
+              value: (event?.target as HTMLInputElement)?.value,
+            }
+          : e,
+      );
+      onChange(newValue);
+      setEditInput(false);
+    };
+
+    return (
+      <Tooltip label={validationMessage ? validationMessage : ''}>
+        <Input
+          autoFocus
+          className='w-auto inline text-warning-700 h-8 hover:border-transparent focus:border-transparent'
```

Comment:
you have on input variant the variant unstyled and should remove the focus + hover effects